contracts/lib/ConfigLib.sol

```diff
@@ -11,16 +11,34 @@ import { HashLib } from "./HashLib.sol";
 import { EnumerableSet } from "../utils/EnumerableSet4337.sol";
 
 library ConfigLib {
+    using FlatBytesLib for FlatBytesLib.Bytes;
     using EnumerableSet for EnumerableSet.AddressSet;
-    using HashLib for *;
+    using EnumerableSet for EnumerableSet.Bytes32Set;
+    using HashLib for string;
+    using AssociatedArrayLib for AssociatedArrayLib.Bytes32Array;
+    using IdLib for PermissionId;
     using ConfigLib for *;
-    using AssociatedArrayLib for *;
-    using IdLib for *;
 
     error UnsupportedPolicy(address policy);
 
-    IRegistry internal constant registry = IRegistry(0x000000000069E2a187AEFFb852bF3cCdC95151B2);
-    ModuleType internal constant POLICY_MODULE_TYPE = ModuleType.wrap(7);
+    function requireSupportsInterface(address policy, PolicyType policyType) internal view {
```

Comment:
im not super sure if I like this.

@filmakarov should make 3 different enable functions for the different policy types, so we dont have to do this in runtime?